pyproject.toml

```diff
@@ -59,7 +61,13 @@ dev = [
     "mistralai>=1",
     "boto3>=1.34.88",
     "scikit-learn==1.4.2",
+    "bert-score>=0.3.13",
```

Comment:
why did you duplicate these dependencies here? Is there a specific reason? I'm not sure if we rely on them directly and they might be included in the other libraries, like langchain already?